libqtile/core/manager.py

```diff
@@ -102,11 +94,22 @@ def __init__(
         self.widgets_map = {}
         self.groups_map = {}
         self.groups = []
+        self.dgroups = None
         self.keys_map = {}
         self.current_chord = False
 
         self.numlock_mask, self.valid_mask = self.core.masks
 
+        self.current_screen = None
+        self.screens = []
+        self._drag = None
+        self.mouse_map = None
+        self._state = state
+        self.server = IPCCommandServer(self)
+
+        self.run_config()
+
+    def run_config(self):
```

Comment:
We really needed something like this, so glad it's here! Maybe change it to `load_config` or `read_config`? I think they might be more clear to a user, and I expect we will soon turn this into a `cmd_` function for live config reloading.

Reply:
I'm gonna see about making another change I was thinking about along with this.

```
self.config = self.load_config(config)
```

however, it's possible that we might have some parts of init outside of qtile that depend on qtile.config being there when libqtile.init is called. It's possible I can reorder this to be MUCH cleaner, but unsure about how much breakage will occur.